src/vs/workbench/services/extensions/browser/webWorkerExtensionHost.ts

```diff
@@ -83,9 +83,10 @@ export class WebWorkerExtensionHost extends Disposable implements IExtensionHost
 		}
 		COI.addSearchParam(suffixSearchParams, true, true);
 
-		const suffix = `?${suffixSearchParams.toString()}`;
+		let suffix = `?${suffixSearchParams.toString()}`;
```

Comment:
I feel like you could just do this:
```suggestion
		if (platform.Language.isDefaultVariant()) {
			suffixSearchParams.set('locale', platform.language);
		}
		const suffix = `?${suffixSearchParams.toString()}`;
```

and get rid of the other changes in the file.